refactor: make `ddev auth ssh` flags mutually exclusive, fixes #6795 (#6796)
ddev/ddev: docs/content/users/usage/commands.md

Add [SSH key authentication](../usage/cli.md#ssh-into-containers) to the `ddev-ssh-agent` container.

> The command can be executed multiple times to add more keys to the same `ssh-agent`.

Reviewer: ```suggestion
The command can be executed multiple times to add more keys, or you can use the `-f` flag for several individual keys for example `ddev ssh add -f /path/to/first_id_rsa -f /path/to/second_id_rsa`.
```

Example:

```shell
# Add your SSH keys to the SSH agent container
ddev auth ssh
# Add your SSH keys from ~/custom/path/to/ssh directory
ddev auth ssh -d ~/custom/path/to/ssh
# Add your SSH keys from ~/.ssh/id_ed25519 and ~/.ssh/id_rsa files
ddev auth ssh -f ~/.ssh/id_ed25519 -f ~/.ssh/id_rsa
```

Flags:

> * `--ssh-key-file`, `-f`: Full path to SSH private key file, use the flag multiple times to add more keys.

Reviewer: Relative path works fine, right?

```suggestion
* `--ssh-key-file`, `-f`: Path to SSH private key file, use the flag multiple times to add more keys.
```

Author: Yes, the relative path works too.

> * `--ssh-key-path`, `-d`: Full path to directory with SSH private key(s), use the flag multiple times to add more directories.

Reviewer: ```suggestion
* `--ssh-key-path`, `-d`: Path to directory with SSH private key(s), use the flag multiple times to add more directories.
```
